docs/schema-update-playbook.md

```diff
@@ -0,0 +1,25 @@
+# Bytebase Schema Update Playbook
+
+All schemas are located at [store/migration](https://github.com/bytebase/bytebase/tree/main/store/migration). Bytebase schema release follows semantic versioning. This document goes over the scheme update process during development and release.
+
+## Development
+We use different schema versions for [dev](https://github.com/bytebase/bytebase/blob/main/bin/server/cmd/profile_dev.go) and [release](https://github.com/bytebase/bytebase/blob/main/bin/server/cmd/profile_release.go). 
+
+If schema versions in dev/release profiles are the same (e.g. 1.1.0),
+1. Create a new directory under [migration directory](https://github.com/bytebase/bytebase/tree/main/store/migration) with the new version, e.g. 1.2.0.
+2. Copy over the [latest.sql](https://github.com/bytebase/bytebase/blob/main/store/migration/1.1.0/latest.sql) schema file and [latest_data.sql](https://github.com/bytebase/bytebase/blob/main/store/migration/1.1.0/latest_data.sql) data file from the previous version. And make a PR for changes above.
+3. Bump up the MINOR version in the [dev schema version](https://github.com/bytebase/bytebase/blob/main/bin/server/cmd/profile_dev.go), e.g. 1.1.0 will be 1.2.0.
+4. Add a new SQL file with DDL statements with a descriptive name, e.g. [0000_github_com.sql](https://github.com/bytebase/bytebase/blob/main/store/migration/1.1.0/0000_github_com.sql).
```

Comment:
0000 will be used as the PATCH version?

Reply:
Nah, I don't feel like it's a patch but the ordering of DDL statements, as we state the PATCH in semantic versioning for data change only.

Reply:
What if we consider repurposing it. This doesn't conflict with the data change case. And it plays with the development workflow nicely. MINOR version is used for the external release, while PATCH version is used during the development cycle to advance the schema change under a MINOR.

Reply:
I think it's more about the version string in the migration history. That was another reason I was trying not to put a number before to reduce the confusion. Maybe we should add a manifest file to specify the DDL file ordering later.